weaviate/templates/weaviateStatefulset.yaml

```diff
@@ -43,7 +45,7 @@ spec:
           privileged: true
         image: "{{ .image.registry }}/{{ .image.repo }}:{{ .image.tag }}"
         imagePullPolicy: "{{ .image.pullPolicy }}"
-        command: ["sysctl", "vm.max_map_count={{ .sysctlVmMaxMapCount }}", "vm.overcommit_memory=1" ]
+        command: ["sysctl", "-w", "vm.max_map_count={{ .sysctlVmMaxMapCount }}", "vm.overcommit_memory=1" ]
```

Comment:
I thought that this was already fixed, no Idea what happened to the fix.

Reply:
```
COMPATIBILITY
     The -w option has been deprecated and is silently ignored.
```
Hmmm

Reply:
FWIW `-w` is deprecated on MacOS, and I checked the sources for ubuntu and Alpine linux and they both have the logic `if (WriteMode || strchr(*argv, '='))` so the `-w` should be redundant.

Reply:
JFYI: it was changed here: https://github.com/weaviate/weaviate-helm/pull/149

It should not work with `-w` in theory and it indeed breaks init container in our deployments.

Reply:
The docs say that it is deprecated and is ignored. @rbtz-openai so why does it break for you?

I can release a new patch without the `-w`.